<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankRegister</class>
 <widget class="QDialog" name="BankRegister">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>162</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Banco</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="topmenu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/png/topright.jpg);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="labelStatus">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>1</y>
        <width>491</width>
        <height>39</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Cadastrar Banco</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>220</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text">
        <string>&amp;Salvar</string>
       </property>
       <property name="icon">
        <iconset resource="../../../icons/resources.qrc">
         <normaloff>:/png/add-icon.png</normaloff>:/png/add-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text">
        <string>S&amp;air</string>
       </property>
       <property name="icon">
        <iconset resource="../../../icons/resources.qrc">
         <normaloff>:/png/shutdown-icon.png</normaloff>:/png/shutdown-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Informações</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Códgo de Compensação</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadLineEditEnter" name="lineEditCode">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome do Banco</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadLineEditEnter" name="lineEditDescription">
          <property name="maxLength">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRadLineEditEnter</class>
   <extends>QLineEdit</extends>
   <header location="global">dsmlineeditenter.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditCode</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
  <include location="../../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
